app/views/shared/_zendesk_snippet.html.erb

```diff
@@ -4,6 +4,6 @@
   <script
     id="ze-snippet"
     nonce="<%= request.content_security_policy_nonce %>"
-    src=https://static.zdassets.com/ekr/snippet.js?key=904d04ae-8027-4ebb-ad39-d0e9f68d6ab1>
+    src=https://static.zdassets.com/ekr/snippet.js?key=3c3c63b0-8490-4d26-91f6-8e88f4be28f3>
```

Comment:
Where do we find this key, can you see it when you login to Zendesk? I couldn't find it.